tmail-backend/apps/postgres/src/main/java/com/linagora/tmail/james/app/PostgresTmailServer.java

```diff
@@ -227,13 +229,13 @@ public static GuiceJamesServer createServer(PostgresTmailConfiguration configura
         new PostgresDeletedMessageVaultModule(),
         new PostgresEventStoreModule());
 
+    public static final Module PLUGINS = new QuotaMailingModule();
     private static final Module POSTGRES_MODULE_AGGREGATE = Modules.override(Modules.combine(
-        new MailetProcessingModule(), new DKIMMailetModule(), POSTGRES_SERVER_MODULE, PROTOCOLS, JMAP_LINAGORA))
```

Comment:
Why removing the DKIMMailetModule ?

Reply:
mistake when rebase resolve conflict 
let me fix it